gov2/cloudwatch/README.md

```diff
@@ -0,0 +1,113 @@
+# AWS SDK for Go V2 code examples for Amazon CloudWatch
+
+## Purpose
+
+These examples demonstrates how to perform several Amazon CloudWatch operations
+using version 2 of the AWS SDK for Go.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the AWS SDK for Go Developer Guide.
+
+## Running the code
+
+### CreateCustomMetric/CreateCustomMetricv2.go
+
+This example creates a new Amazon CloudWatch metric in a namespace.
+
+`go run CreateCustomMetricv2.go -n NAMESPACE -m METRIC-NAME -s SECONDS -dn DIMENSION-NAME -dv DIMENSION-VALUE`
+
+- _NAMESPACE_ is the namespace for the metric.
+- _METRIC-NAME_ is the name of the metric.
+- _SECONDS_ is the number of seconds for the metric.
+- _DIMENSION-NAME_ is the name of the dimension.
+- _DIMENSION-VALUE_ is the value of the dimension.
+
+The unit test accepts similar values in _config.json_.
+
+### CreateEnableMetricAlarm/CreateEnableMetricAlarmv2.go
+
+This example enables the specified Amazon CloudWatch alarm.
+
+`go run CreateEnableMetricAlarmv2.go -n INSTANCE-NAME -i INSTANCE-ID -a ALARM-NAME`
+
+- _INSTANCE-NAME_ is the name of the EC2 instance for which the alarm is enabled.
```

Comment:
'Amazon Elastic Compute Cloud (Amazon EC2)' on first mention in a code file.

Reply:
Fixed